TypeCobol/Compiler/Sql/Model/SelectClause.cs

```diff
@@ -40,12 +40,12 @@ public class UnpackedRowSelection : Selection
         public override SelectionType Type => SelectionType.UnpackedRow;
     }
 
-    public class DotStarSelection : Selection
+    public class TableOrViewAllColumnsSelection : Selection
     {
         public SymbolReference TableOrViewOrCorrelationName { get; }
         public override SelectionType Type => SelectionType.DotStar;
```

Comment:
Rename the enum also to `TableOrViewAllColumns`.

Reply:
The g4 grammar must be updated also to reflect the renaming. Do a global text search to locate the parts that need to be renamed.